plugins/postcss-double-position-gradients/src/has-supports-at-rule-ancestor.js

```diff
@@ -0,0 +1,39 @@
+export function hasSupportsAtRuleAncestor(node) {
+	let parent = node.parent;
+	while (parent) {
+		if (parent.type !== 'atrule') {
+			parent = parent.parent;
+			continue;
+		}
+
+		if (parent.name === 'supports') {
+			if (parent.params.indexOf('conic-gradient(') !== -1) {
+				return true;
+			}
+
+			if (parent.params.indexOf('linear-gradient(') !== -1) {
+				return true;
+			}
+
+			if (parent.params.indexOf('radial-gradient(') !== -1) {
+				return true;
+			}
+
+			if (parent.params.indexOf('repeating-conic-gradient(') !== -1) {
+				return true;
+			}
+
+			if (parent.params.indexOf('repeating-linear-gradient(') !== -1) {
+				return true;
+			}
+
+			if (parent.params.indexOf('repeating-radial-gradient(') !== -1) {
```

Comment:
A bit [nit], should we use `includes` for clarity?
Furthermore, should we abstract this a bit given that this is what is being tested on `gradientRegExp` and `gradientPartsRegExp`? Fine if not!

Reply:
I avoided the `regexp` on purpose as I completely missed that this plugin also does more than `linear-gradient` and `conic-gradient`.

There were no tests for the other 4 functions, no mention in the README.md and the regexp does have these, but it's not immediately obvious.